react/week4/my-app/src/components/RenderProps.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState, useEffect } from "react";
+import RepoList from "./RepoList";
+
+function Repos() {
+  const [list, setList] = useState([]);
+  useEffect(() => {
+    const URL = `https://api.github.com/users/hemanth06912/repos`;
+    (async () => {
+      try {
+        const response = await fetch(URL);
+        const json = await response.json();
+        const result = json;
```

Comment:
Any reason for introducing one more const here? You could directly pass `json` to `setList`